lib/parallel_tests/rspec/runner.rb

```diff
@@ -18,7 +18,10 @@ def determine_executable
           when File.exist?("bin/rspec")
             ParallelTests.with_ruby_binary("bin/rspec")
           when ParallelTests.bundler_enabled?
-            cmd = (run("bundle show rspec-core") =~ %r{Could not find gem.*} ? "spec" : "rspec")
+            installed_version = Gem::Version.new(run("bundle -v").gsub(/\A\D+/, ""))
```

Comment:
shelling out is kinda slow/brittle ...
can we use `Bundler::VERSION` ?
... or  do a `require 'bundler/version'` i it is not defined